app/src/main/java/com/pvp/app/ui/screen/task/TaskScreen.kt

```diff
@@ -15,4 +44,116 @@ fun TaskScreen(
+            ) {
+            Checkbox(
+                checked = supportsDistanceMetrics,
+                onCheckedChange = { supportsDistanceMetrics = it },
+                modifier = Modifier.padding(bottom = 16.dp)
+            )
+            Text (
+                text = "Supports distance metrics",
+                modifier = Modifier.padding(start = 16.dp)
+            )}
+
+            TextField(
+                value = title,
+                onValueChange = { title = it },
+                label = { Text("Title") },
+                modifier = Modifier.fillMaxWidth().padding(bottom = 16.dp)
+            )
+
+            TextField(
+                value = duration,
+                onValueChange = { duration = it },
+                label = { Text("Duration") },
+                keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Number),
+                modifier = Modifier.fillMaxWidth().padding(bottom = 16.dp)
+            )
+
+            DatePicker(
+                selectedDate = startDate,
+                modifier = Modifier.fillMaxWidth().padding(bottom = 16.dp)
+            )
+
+            Button(onClick = { /* ???? */ }) {
+                Text("Submit")
+            }
+        }
+    }
+}
+
+@Composable
+fun DatePicker(
+    selectedDate: Date,
+    modifier: Modifier = Modifier
+) {
+    val calendar = Calendar.getInstance()
+    calendar.time = selectedDate
+    val year = calendar.get(Calendar.YEAR)
+    val month = calendar.get(Calendar.MONTH)
+    val day = calendar.get(Calendar.DAY_OF_MONTH)
+
+    Column(modifier) {
+        Text("Start Date")
+        Spacer(modifier = Modifier.height(8.dp))
+        Row(verticalAlignment = Alignment.CenterVertically) {
+            Text("Year:")
+            Spacer(modifier = Modifier.width(8.dp))
+            OutlinedTextField(
+                value = year.toString(),
+                onValueChange = { },
+                keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Number),
+                modifier = Modifier.weight(1f)
+            )
+            Spacer(modifier = Modifier.width(16.dp))
+            Text("Month:")
+            Spacer(modifier = Modifier.width(8.dp))
+            OutlinedTextField(
+                value = (month + 1).toString(),
+                onValueChange = { },
+                keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Number),
+                modifier = Modifier.weight(1f)
+            )
+            Spacer(modifier = Modifier.width(16.dp))
+            Text("Day:")
+            Spacer(modifier = Modifier.width(8.dp))
+            OutlinedTextField(
+                value = day.toString(),
+                onValueChange = { },
+                keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Number),
+                modifier = Modifier.weight(1f)
+            )
+        }
+    }
+}
+
+
+@Preview(showBackground = true)
+@Composable
+fun DefaultPreview() {
+    SportTaskForm()
```

Comment:
I have a feeling that this component is unused. If that is the case (and it is only used for previewing purposes), then it should be removed before executing a merge. You can preview `SportTaskForm` component by annotating it with `@Preview` annotation. This annotation can be given to multiple components within the same file, all should be rendered in the IDE side just fine.

Also, at the end of the file a single empty line _could_ be added for consistency. That's how UNIX file endings work.